src/science/canopy/cbl_dryLeaf.F90

```diff
@@ -369,15 +381,8 @@ SUBROUTINE dryLeaf( dels, rad, rough, air, met,                                &
                      rdx(i,2) = rdx(i,2) * &
                      (0.5 - 0.05*LOG(jtomol*1.0e6*rad%qcan(i,1,2)))
 
-!$                xleuning(i,1) = ( fwsoil(i) / ( csx(i,1) - co2cp3 ) )              &
-!$                     * ( veg%a1gs(i) / ( 1.0 + dsx(i)/veg%d0gs(i)))
-!$                xleuning(i,2) = ( fwsoil(i) / ( csx(i,2) - co2cp3 ) )              &
```

Comment:
This commenting style has lead to issues in some UM builds. Its redundant code anyway